logdna/logdna.py

```diff
@@ -169,7 +171,6 @@ def close(self):
 
         Make sure that the log handler has attempted to flush the log buffer before closing.
         """
-        try:
+        if self.exception_flag == False and len(self.buf) > 0:
```

Comment:
`if not self.exception_flag and len(self.buf) > 0:`

I'd actually reverse the conditions to be consistent with line 78.